src/com_tjnotifications/admin/views/notifications/tmpl/default.php

```diff
@@ -182,6 +189,58 @@
 									}
 									?>
 								</td>
+								
+								<td class = "d-md-table-cell">
```

Comment:
also make the code generic, not provider specific

Reply:
Sir these need to be stored separately so i added two different array for storing hence they need to be displayed separately.